transcendence-app/package.json

```diff
@@ -68,9 +68,7 @@
     "@types/cookie-parser": "^1.4.3",
     "@types/express": "^4.17.14",
     "@types/express-session": "^1.17.5",
-    "@types/jest": "29.2.4",
     "@types/multer": "^1.4.7",
```

Comment:
Al of these changes are made in order to eventually have `"pong-engine": "^1.0.0",` this here, isn't it? That you will place in a follow up PR?

Reply:
Yes! It's being used in the webapp indeed -> https://github.com/42AGV/ft_transcendence/blob/7d59b0e78c136476b7a8c6f2f1bde5d1247b08cf/webapp/package.json#L7

and in the follow up PR this will be also imported in the backend